src/internal/utils.js

```diff
@@ -148,3 +148,24 @@ export function wrapSagaDispatch(dispatch) {
     return dispatch(wrappedAction)
   }
 }
+
+/**
+ * takes a generator and return a wrapped generator that you can clone.
+ */
+export const cloneableGenerator = (generatorFunction) => (...args) => {
```

Comment:
`generatorFunction` feels verbose, probably just `generator` would be ok

also thinking if this should be just called `cloneable`, but pick whichever you want for now